client/src/components/Loading/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import { CircularProgress } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    marginTop: '14%',
+    color: theme.palette.primary,
+  },
+}));
+
+const LoadingProgressBar = () => {
+  const classes = useStyles();
+
+  return (
+    <div className={classes.root}>
+      <CircularProgress size={70} thickness={4} />
```

Comment:
I think you will want this LoadingProgressBar component to take a `value` prop and then pass that `value` into the `CircularProgress` here.  But of course that can be a next step, when you are ready to use this component in the real app.